### PR TITLE
feat: expose globals edit

### PR DESCRIPTION
**Pull request by tallpress**
Changes:

- Exposes globals_edit to Eva.
- Fixes failing TestEva_API test by using requests_mock decorator
- Adds note about failing test_get

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,14 +4,14 @@ The Eva Python SDK provides convenient access to the Automata Eva API from appli
 
 __* This SDK is currently in beta__
 
-- [Eva Python SDK](#eva-python-sdk)
+- [Eva Python SDK ![PyPI version](https://badge.fury.io/py/evasdk) ![Build status](https://github.com/automata-tech/eva_python_sdk/workflows/Build%20and%20test/badge.svg) [![codecov](https://codecov.io/gh/automata-tech/eva_python_sdk/branch/development/graph/badge.svg)](https://codecov.io/gh/automata-tech/eva_python_sdk)](#eva-python-sdk---)
```
Comment (tallpress) [inline README.md]: not sure where this came from, feel like this is the new bade or something, can remove if this is an issue

File: README.md [hunk 2]
```diff
@@ -4,14 +4,14 @@ The Eva Python SDK provides convenient access to the Automata Eva API from appli
...
   - [Installation](#installation)
     - [Pip](#pip)
     - [Pipenv](#pipenv)
     - [Detail Instructions](#detail-instructions)
   - [Examples](#examples)
     - [Eva](#eva)
-    - [evasdk.eva_http and evasdk.eva_ws](#evasdkevahttp-and-evasdkevaws)
+    - [evasdk.eva_http and evasdk.eva_ws](#evasdkeva_http-and-evasdkeva_ws)
```
Comment (tallpress) [inline README.md:14]: same with this, auto generated

File: README.md (hunk 1 above)
Comment (LouisBrunner) [reply README.md]: It would be good to see where this is coming from, it looks like a copy from the title?

**Review by LouisBrunner — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tallpress) [reply README.md]: yeah, does on autosave, will remove

File: README.md (hunk 2 above)
Comment (tallpress) [reply README.md:14]: @LouisBrunner you want me to remove this diff?

**Review by LouisBrunner — APPROVED** [review]

**Pull request merged**